src/components/Friends.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+export default class Friends extends Component {
+
+	render(){
+		return(
+			<div className="column">
+				<h1>List of Friends</h1>
+				<ul>
+					{this.props.friendList.map(friend => <li key={this.props.friendList.indexOf(friend)}>{friend.first} {friend.last} </li>)}
```

Comment:
You could use the second param from map instead of looking up the index here! `.map(friend, i) =>`